tests/pytests/functional/modules/test_win_file.py

```diff
@@ -0,0 +1,20 @@
+import os
+import pytest
+from salt.modules import win_file
+
+
+pytestmark = [
+    pytest.mark.skip_on_windows,
+]
```

Comment:
```suggestion
```

This seems to be a windows specific test, we don't want to skip that on windows.